src/content/core/okteto-insights-dashboards.mdx

```diff
@@ -0,0 +1,99 @@
+
+
+<p align="center">
+  <Image
+    src={require("@site/static/img/insights-pods.jpg").default}
+    alt="Okteto Insights Dashboard Example"
+    width="800"
+  />
+</p>
+
+## Insights Dashboards
+
+**Dashboards available:**
+- [Node Metrics Dashboard](#node-metrics-dashboard)
+- [Pod Metrics Dashboard](#pod-metrics-dashboard)
+- [Build and Deploy Metrics Dashboard](#build-and-deploy-metrics-dashboard)
+- [Namespace, Preview, and User Metrics Dashboard](#namespace-preview-and-user-metrics)
+
+### Node Metrics Dashboard
+The Node Metrics dashboard is designed to give you a high level overview of cluster health. Focusing on CPU and Memory utilization, this can help you determine if your dev cluster has resources over-provisioned or under-provisioned.
+
+Variables:
+- Node Name: The name of the Kubernetes node to display specific data for.
+
+  <Image
+    src={require("@site/static/img/insights-dashboard-node.png").default}
+    alt="Okteto Insights Dashboard Example"
+    width="800"
+  />
+
+### Pod Metrics Dashboard
+The Pod Metrics dashboard shows lower level information related to each Kubernetes Pod. This can help you to analyze the performance (memory leaks, CPU optimization, etc.) of the application code running in a particular namespace.
+
+Variables
+- Namespace: The name of the Okteto Namespace to display specific data for, filtered by the selected Development Environment
+- Development Environment: The specific Okteto Development environment to display data for.
+
+  <Image
+    src={require("@site/static/img/insights-dashboard-pod.png").default}
+    alt="Okteto Insights Dashboard Example"
+    width="800"
+  />
+
+### Build and Deploy Metrics Dashboard
+The Build and Deploy Metrics dashboard helps you understand how your application has been evolving over time by giving you data related to build and deploy times.
+
+Variables
+- Development Environment: Representing the name of the git repository to display specific metrics for. You can choose to see these metrics for the whole cluster or a particular application you have.
+
+  <Image
+    src={require("@site/static/img/insights-dashboard-build-deploy.png").default}
+    alt="Okteto Insights Dashboard Example"
+    width="800"
+  />
+
+### Namespace, Preview, and User Metrics 
+The Namespaces, Previews, and Users dashboard gives you metrics which help you measure how the platform is being adopted and used across your organization.
+
+  <Image
+    src={require("@site/static/img/insights-dashboard-namespace-preview-user.png").default}
+    alt="Okteto Insights Dashboard Example"
+    width="800"
+  />
+
+:::tip
+If you're interested in the calculations behind these dashboards, we've [open-sourced them](https://github.com/okteto/insights-dashboards) based on the Okteto Insights [data made available](#what-data-is-available) through our prometheus endpoint.
+:::
+
+## What data is available
```

Comment:
I would remove this paragraph. This page is only about the dashboard. I think [this callout](https://github.com/okteto/docs/pull/858/files#diff-ee5c15f1c8dbfc7b8f0cd89a2debeb2a46c7a4d09d2d9be77f88de23529d3908R95) is great if people want to know more